javascript/nanoexpress/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress": "^1.1.4"
```

Comment:
could you use `~1.1.4` to keep in `1.1` ?

Reply:
I fixed correctly? I want use always use latest ver in benchmarks